CSharp_Course_Project/lib.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Course_Project
+{
+    class lib
```

Comment:
It is very good that you started to use classes in separate files.
But you should remember and follow the code guidelines.
Class name should be started with capital letter and should have clear name.